src/Microsoft.Health.Fhir.Api/Resources.resx

```diff
@@ -123,6 +123,9 @@
   <data name="ContentTypeHeaderRequired" xml:space="preserve">
     <value>The "content-type" header is required.</value>
   </data>
+  <data name="CustomAuditHeaderTooLarge" xml:space="preserve">
+    <value>The size of the header {0} is {1}.  The maximum allowed size is {2}.  Please retry the request with a smaller value.</value>
```

Comment:
The message here and below should be a bit more consistent. Maybe..

The maximum length of a custom audit header value is {0}. The supplied custom audit header '{1}' has length of {2}.
The maximum number of custom audit headers allowed is {0}. The number of custom audit headers supplied is {1}.